web_src/js/features/notification.ts

```diff
@@ -33,8 +33,7 @@ export function initNotificationsTable() {
 
 async function receiveUpdateCount(event: MessageEvent) {
   try {
-    const data = JSON.parse(event.data);
-
+    const data = event.data;
```

Comment:
It seems that the SSE has been broken for long time, no one ever noticed or complained ....

Reply:
Probably most people haven't been aware the feature even existed in the first place

Might be something to test using e2e at some point 😅

Reply:
Unfortunately, no progress on it (and no people), for example: E2e Test Fixes #25058